website/content/docs/secrets/pki/est.mdx

```diff
@@ -0,0 +1,181 @@
+be configured to return [batch tokens](/vault/docs/concepts/tokens#batch-tokens).
+Batch tokens are required to avoid lease explosions as every EST incoming
+request needs to be authenticated.
+
+The path within an ACL policy, must match the internal redirected path including
+the mount and not the `.well-known/est/` URI the client is initially using.
+The path to use within the plugin depends on the path policy that is in configured
+for the EST label being used by the client.
+
+If using the `sign-verbatim` as a path policy, the following
+ACL policy will allow an authenticated client access the required PKI EST paths.
+```
+path “pki/est/simpleenroll” {
+  capabilities=[“update”, “create”]
+}
+path “pki/est/simplereenroll” {
+  capabilities=[“update”, “create”]
+}
+```
+
+For a role base path policy, this sample policy can be used
+```
+path “pki/roles/my-role-name/est/simpleenroll” {
+  capabilities=[“update”, “create”]
+}
+path “pki/roles/my-role-name/est/simplereenroll” {
+  capabilities=[“update”, “create”]
+}
+```
+
+#### Updating the PKI mount tunable parameters
+
+Once the authentication mount has been created and configured, the authentication mount's accessor
+will need to be captured and added within the PKI mount's [delegated auth accessors](/vault/api-docs/system/mounts#delegated_auth_accessors).
+
+To get an authentication mount's accessor field, the following command can be used.
+
+```shell-session
+$ vault read -field=accessor sys/auth/auth/userpass
+```
+
+For EST to work within certain clients, a few response headers need to be explicitly allowed
+along with configuring the list of accessors the mount can delegate authentication towards.
+The following will grant the required response headers, you will need to replace the values for the `delegated-auth-accessors`
+to match your values.
+
+```shell-session
+$ vault secrets tune \
+  -allowed-response-headers="Content-Transfer-Encoding" \
+  -allowed-response-headers="Content-Length" \
+  -allowed-response-headers="WWW-Authenticate" \
+  -delegated-auth-accessors="auth_userpass_e2f4f6d5" \
+  -delegated-auth-accessors="auth_cert_4088ac2d" \
+  pki
+```
+
+#### PKI EST configuration
+
+The EST protocol specifies that an EST server must support a URI path-prefix of
+`.well-known/est/` as defined in [RFC-5785](https://datatracker.ietf.org/doc/html/rfc5785).
+EST client's won't provide any sort of configuration for paths, and will
+default to hitting the host on the path `https://<hostname>:<port>/.well-known/est/`.
+
+Some clients allow a single label, sometimes referred to as `additional path segment`,
+to accommodate different issuers. This label will be added to the path after
+the est path such as `https://<hostname>:<port>/.well-known/est/<label>/`.
+
+To provide different restrictions around usage (defaults, an issuer or role),
+for EST protocol endpoints, a path policy is associated with the EST
+label.
+
+@include 'pki-est-default-policy.mdx'
```

Comment:
This introduces the internal redirected paths, before explaining the redirection.  I double-checked with a friend, but this isn't at all clear reading through.  Something that would really help is a table explaining the redirection before a table explaining the possible things to redirect to.

The pki path here doesn't really make sense - why would the person configuring this need to know about this path?I think it would be helpful here to show the EST path, alongside the PKI-path.

That could be added to this - though I realize that would require many more entries, but perhaps a separate table would be helpful: 

| EST endpoint | Chooses PKI Mount By | Policy |
| `/.wellknown/est/{cacerts/simpleenroll/simplereenroll} | `is_default` is true | `default_path_policy` |
| `/.wellknown/est/<label>/{...}` | `label_to_path_policy` matching `label` | `label_to_path_policy` mapping |

Then show the policy to path:

| Policy (`default_path_policy` or `label_to_path_policy` mapping) | Maps to |
| 'sign-verbatim' | `/<mount>/est/{cacerts/simpleenrolls/simplereenroll}` |
| 'role:<role>' | `/<mount>/roles/<role>/est/{cacerts/simplenroll/simplreenroll}` |